nupic/research/frameworks/vernon/distributed/experiments/cl_experiment.py

```diff
@@ -0,0 +1,85 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+from torch.nn import DataParallel
+from torch.nn.parallel import DistributedDataParallel
+
+from nupic.research.frameworks.pytorch.dataset_utils.samplers import (
+    TaskDistributedSampler,
+)
+from nupic.research.frameworks.vernon.distributed.experiments.components import (
+    DistributedBase,
+)
+from nupic.research.frameworks.vernon.experiments.cl_experiment import (
+    ContinualLearningExperiment as ContinualLearningExperimentBase,
+)
+
+__all__ = [
+    "ContinualLearningExperiment",
+]
+
+
+class ContinualLearningExperiment(DistributedBase,
```

Comment:
AFAIK we don't we have any tests for ContinualLearning experiment yet, neither unit or regression. That is not the subject of this PR, but just a note that while we don't have tests we can unintentionally break some code with something as simple as a typo in a refactoring. 

I can test it later after you merge to make sure that is not the case (and probably see if I write some unit tests in the process).

Reply:
For what it's worth, I have been running the CL and Meta CL experiments to make sure they don't throw exceptions. But I haven't been verifying correct results.